sql/parser/eval.go

```diff
@@ -256,6 +304,16 @@ func (d DTuple) Next() Datum {
 	return n
 }
 
+// IsMax implements the Datum interface.
+func (d DTuple) IsMax() bool {
+	return false
+}
+
+// IsMin implements the Datum interface.
+func (d DTuple) IsMin() bool {
+	return false
+}
```

Comment:
It seems that these `false`s above are driven by the way you're using `Is{Min,Max}` (at least otherwise, an empty tuple would seem "minimal" to me in some way).
Could you add a comment on the interface explaining what is expected of the implementation?